google/cloud/sql/connector/connector.py

```diff
@@ -44,4 +49,8 @@ def connect(instance_connection_string, **kwargs):
     #
     # Return a DBAPI connection
 
-    raise NotImplementedError()
+    loop = asyncio.new_event_loop()
+    thr = Thread(target=loop.run_forever)
```

Comment:
This `loop`/`thread` combo would be a good target for a fixture.

```python
@fixture
def async_loop():
     loop = asyncio.new_event_loop()
     thread = Thread(target=loop.run_forever)
     thread.start()
     yield loop
     loop.stop()
     thread.join()
```

Reply:
The fixture would be in the tests, right/

Reply:
Yes - sorry that wasn't clear.